string-manipulation-2/src/swap-chars.js

```diff
@@ -0,0 +1,14 @@
+/* exported swapChars */
+function swapChars(firstIndex, secondIndex, string) {
+  var array = string.split('');
+  array.splice(firstIndex, 1, string[secondIndex]);
+  array.splice(secondIndex, 1, string[firstIndex]);
+  return array.join('');
```

Comment:
You should try to think of a way to do this without turning the input string into an array. Each method you called is doing an entire loop over the data, so this function has four loops within it despite being short. If you do this yourself, you can do it in one loop 🤓